custom-namespaces.yaml

```diff
@@ -1028,16 +1028,29 @@ data_integrity_monitoring:
   views:
     run_history:
       tables:
-        - table: data-monitoring-dev.dim.dim_run_history
+        - table: data-monitoring-dev.dim_external.run_history
       type: table_view
     run_processing_info:
       tables:
-        - table: data-monitoring-dev.dim.dim_run_processing_info
+        - table: data-monitoring-dev.dim_external.run_processing_info
```

Comment:
LGTM.
You may want to use the standard for datasets that we use in moz-fx-data-shared-prod:
e.g. dim_derived (to store tables) > dim (to store views)

Reply:
hmm, I thought about it but `_derived` would also not be correct in this case as this table is populated by the application itself. Could have always called it something like `dim_operational`, but we can always do that if we ever decide to invest more time into it.